### PR TITLE
Patients CRUD Blade

### PR DESCRIPTION
**Pull request by A-Mi-Mi-Htun**
## Link to Issue
task08: Patients CRUD

## What you did
Patient CRUD blade

## What not to do
There's none.

## Operation check
Check in browser: /patients/list
patient list screen will show.
Check in browser: /patients/create
patient create screen will show.
Check in browser: /patients/edit
patient edit screen will show.
Check in browser: /patients/show
patient show screen will show.
Check in browser: /patients/login
patient login screen will show.

## Evidence
image
image
image
image
image

## Other
There's none.

### CODE REVIEW

File: resources/views/patients/list.blade.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+                        <td><img src="{{ asset('img/img_user.png') }}" class="listImg" style="width: 35px; height: 35px;" alt="profile"></td>
+                        <td>Patient One</td>
+                        <td>patientone@gmail.com</td>
+                        <td>09123456789</td>
+                    </tr>
+                </table>
+            </div>
+        </div>
+    </div>
+@endsection
```
Comment (myintmawyee20) [inline resources/views/patients/list.blade.php]: add new line

File: resources/views/patients/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
+    <meta name="description" content="" />
+    <meta name="author" content="" />
+    <title>Hope Clinic</title>
+    <link href="https://cdn.jsdelivr.net/npm/simple-datatables@latest/dist/style.css" rel="stylesheet" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/5.1.3/css/bootstrap.min.css">
+    <link rel="stylesheet" href="https://cdn.datatables.net/1.11.5/css/dataTables.bootstrap5.min.css">
+    <script src="https://use.fontawesome.com/releases/v6.1.0/js/all.js" crossorigin="anonymous"></script>
```
Comment (myintmawyee20) [inline resources/views/patients/show.blade.php]: patient atwet so yin layouts ka patient.blade.php ko extend yuu yin d link twet ka chate fo ma lol top pr buu

File: resources/views/patients/show.blade.php [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+        </div>
+    </footer>
+
+    <!--/.footer -->
+
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js" crossorigin="anonymous">
+    </script>
+    <script src="https://code.jquery.com/jquery-3.5.1.js"></script>
+    <script src="https://cdn.datatables.net/1.11.5/js/jquery.dataTables.min.js"></script>
+    <script src="https://cdn.datatables.net/1.11.5/js/dataTables.bootstrap5.min.js"></script>
```
Comment (myintmawyee20) [inline resources/views/patients/show.blade.php]: patient atwet so yin layouts ka patient.blade.php ko extend yuu yin d link twet ka chate fo ma lol top pr buu

**Review by myintmawyee20 — APPROVED** [review]

File: resources/views/patients/list.blade.php (hunk 1 above)
Comment (A-Mi-Mi-Htun) [reply resources/views/patients/list.blade.php]: htae p br p ama

File: database/factories/PatientFactory.php [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+     * Define the model's default state.
+     *
+     * @return array
+     */
+    public function definition()
+    {
+        return [
+            'name' => $this->faker->name(),
+            'email' => $this->faker->email(),
+            'password' => $this->faker->password(),
```
Comment (YanKyawMinGIT) [inline database/factories/PatientFactory.php]: Pls, use known password!
sample : 
```
use Illuminate\Support\Facades\Hash;

'password' => Hash::make('batch06gp2'),
```

File: database/migrations/2023_01_11_053354_create_patients_table.php [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('patients', function (Blueprint $table) {
+            $table->bigIncrements('id')->comment('id');
+            $table->string('name')->comment('name');
+            $table->string('email')->unique()->comment('email');
+            $table->string('password')->unique()->comment('password');
```
Comment (YanKyawMinGIT) [inline database/migrations/2023_01_11_053354_create_patients_table.php]: sorry for my master excel, I don't want to add unique() in password col. pls, fix it.

File: app/Dao/Patient/PatientDao.php [hunk 6]
```diff
@@ -2,12 +2,33 @@
...
+        return $patients;
+    }
+
+    /**
+     * To show patient by id
+     * @return View patients
+     */
+    public function show($id)
+    {
+        $patient = Patient::find($id);
```
Comment (YanKyawMinGIT) [inline app/Dao/Patient/PatientDao.php]: If param-id not match in Patient id, show 404 page.
sample code :
`$patient = Patient::findOrFail($id);`
OR,
`$patient = Patient::where('id', $id)->firstOrFail();`

**Pull request merged**